Assets/Code/Scripts/TimerTmp.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+
+public class TimerTmp : MonoBehaviour
+{
+    private GameManager gameManager;
+    public TextMeshPro time;
+
+    void Start()
+    {
+        gameManager = FindObjectOfType<GameManager>();
+    }
+
+    void Update()
+    {
+        int secondsElapsed = gameManager.SecondsElapsed;
+        time.text = secondsElapsed.ToString();
```

Comment:
Tę funkcję warto wrzucić bezpośrednio do game managera, raczej nie będzie ona robudowywana, a mimo wszystko jeden plik więcej który może się zgubić

Reply:
Dodatkowo odświeżane jest to co klatkę gdzie wystarczy że wrzucimy to do funkcji inkrementacji timera, rozładuje to obciążenie gry

Reply:
klasa usunięta, metody wrzucone do GameManager.